src/components/GlobalAudioPlayer/MediaController.tsx

```diff
@@ -0,0 +1,80 @@
+'use client'
+
+import React from 'react'
+import { useAccount } from 'wagmi'
+
+import TimeSliderControls from './TimeSliderControls'
+import VolumeControls from './VolumeControls'
+import AudioPlayerControls from './AudioPlayerControls'
+import { useMediaController } from '@/hooks/useMediaController'
+import { Icons } from '@/components/Icons'
+
+export const MediaController: React.FC = () => {
+  const { isConnected } = useAccount()
+  const {
+    currentMedia,
+    medias,
+    isPlaying,
+    playStatus,
+    isMuted,
+    audioRef,
+    handlePlayPause,
+    handleNext,
+    handleBack,
+    setPlayStatus,
+    handleVolumeChange,
+    handleSliderChange,
+    handleAudioMute
+  } = useMediaController()
+
+  if (!isConnected) {
+    return null
+  }
```

Comment:
never return empty components.

instead, conditionally render this component in the parent component